x-pack/solutions/security/plugins/security_solution_serverless/common/pli/pli_config.ts

```diff
@@ -59,12 +62,14 @@ export const PLI_PRODUCT_FEATURES: PliProductFeatures = {
       ProductFeatureKey.endpointPolicyProtections,
       ProductFeatureKey.endpointArtifactManagement,
       ProductFeatureKey.endpointExceptions,
+      ProductFeatureKey.endpointHostIsolation,
     ],
     complete: [
       ProductFeatureKey.endpointPolicyProtections,
       ProductFeatureKey.endpointArtifactManagement,
       ProductFeatureKey.endpointExceptions,
       ProductFeatureKey.endpointHostIsolationExceptions,
+      ProductFeatureKey.endpointHostIsolation,
```

Comment:
Why are they duplicated in the `ProductLine.endpoint` add-on? They are already included in the `ProductLine.security` base PLI.